packages/test-utils/src/convert-annotated-source-to-failure-case.ts

```diff
@@ -52,7 +54,12 @@ type Message<TMessageIds extends string> = {
 type SingleErrorOptions<
   TMessageIds extends string,
   Options extends readonly unknown[],
-> = BaseErrorOptions<Options> & Message<TMessageIds>;
+> = BaseErrorOptions<Options> &
+  Message<TMessageIds> & {
+    readonly languageOptions?: TestLanguageOptions;
+  };
+
+type TestLanguageOptions = ValidTestCase<[]>['languageOptions'];
```

Comment:
ℹ️ This can be removed once https://github.com/typescript-eslint/typescript-eslint/issues/10866 is fixed.